.github/workflows/ci.yml

```
@@ -21,6 +26,7 @@ concurrency:
 
 jobs:
   windows:
+    if: '!github.event.repository.fork'
```

So I'm not sure if we should prevent the CI jobs to run in forks to be honest, they should not have any problem running there.

---

Hm, can we make tests like this dependent on an environment variable? I worry about forks running anything without the user being explicitly aware. We can do something like:
```suggestion
    if: >-
      !github.event.repository.fork
      || fromJSON(env.CONDA_GHA_ENABLED || "false")
```

---

Yeah I understand where you come from, I think we should investigate what other projects that use GitHub Actions use are doing before committing to one solution. I'm also worried we'll forget to add the checks when we evolve the workflows in the future and then end up with partial workflows run in users' namespaces.

---

Fully agree on forgetting 😄, ok will look around at other projects.

---

I looked around and other projects don't seem to care about this at all for _any_ of their workflows 🤷🏻‍♂️

So to avoid overkill I'll only apply the fork check on workflows that deal with conda org specific things (e.g. project boards, etc)